### DISCUSSION TITLE
Save as plain txt?

### DISCUSSION THREAD

**Discussion by Himb000** [LostRuins/koboldcpp#1051] [Ideas]
It would be nice to have an option to save the whole kobold chat into a readable normal txt file.

"
User: Query 1

AI: Response 1

User: Query 2

AI: Response 2
"

etc

**Comment by LostRuins** [comment]
You can do that, click Save -> Share -> Export as Plaintext

**Reply by Himb000** [reply]
oh damn i never saw that. THX!
